sale_commission_pricelist/models/sale_order.py

```diff
@@ -26,13 +26,12 @@ def _get_commission_from_pricelist(self):
     def _onchange_product_id_sale_commission_pricelist(self):
```

Comment:
This should be replaced by a new `depends` on the compute field of `agent_ids` to `order_id.pricelist_id`.

Reply:
@pedrobaeza Done